runtime/doc/lua.txt

```diff
@@ -678,10 +687,16 @@ supported match predicate is `eq?` (both comparing a capture against a string
 and two captures against each other).
 
 vim.treesitter.parse_query(lang, query)
-						*vim.treesitter.parse_query(()*
+						*vim.treesitter.parse_query()*
 	Parse the query as a string. (If the query is in a file, the caller
         should read the contents into a string before calling).
 
+vim.treesitter.get_query(lang, query_name)
```

Comment:
These usually go like `vim.treesitter.get_query({lang}, {query_name})`

And then below you can reference them the same way to make them look parameter style for the docs.